lib/dma.c

```diff
@@ -178,21 +178,32 @@ MOCK_DEFINE(dma_controller_remove_region)(dma_controller_t *dma,
 }
 
 void
-dma_controller_remove_regions(dma_controller_t *dma)
+dma_controller_remove_regions(dma_controller_t *dma,
```

Comment:
I think we should rename this func to dma_controller_remove_all_regions()
This name really confuses with dma_controller_remove_region()

Reply:
sure, done.